apps/extension/public/manifest.json

```diff
@@ -0,0 +1,46 @@
+{
+    "manifest_version": 3,
+    "name": "Wishify Extension",
+    "version": "1.0.0",
+    "action": {
+        "default_popup": "index.html"
+    },
+    "permissions": [
+        "activeTab",
+        "alarms",
+        "background",
+        "bookmarks",
+        "browsingData",
+        "clipboardRead",
+        "clipboardWrite",
+        "contentSettings",
+        "contextMenus",
+        "cookies",
+        "debugger",
+        "declarativeContent",
+        "declarativeNetRequest",
+        "declarativeNetRequestWithHostAccess",
+        "downloads",
+        "history",
+        "identity",
+        "idle",
+        "management",
+        "nativeMessaging",
+        "notifications",
+        "pageCapture",
+        "scripting",
+        "sessions",
+        "storage",
+        "system.cpu",
+        "system.display",
+        "system.memory",
+        "system.storage",
+        "tabCapture",
+        "tabGroups",
+        "tabs",
+        "topSites",
+        "webNavigation",
+        "webRequest",
+        "webRequestBlocking"
+      ]
+}
```

Comment:
We'll remove these as permissions we find out what we actually do need. Leaving them for convenience for now.

Reply:
yeah lol no idea what we need